.env.example

```diff
@@ -1 +1,2 @@
-VUE_APP_WEBSOCKET_PROVIDER=ws://localhost:1234
\ No newline at end of file
+VUE_APP_COLLABORATIVE_ENABLED=false
```

Comment:
As bey default we'll enable the collaborative mode, please set this variable to true in the example